website/content/docs/job-specification/parameterized.mdx

```diff
@@ -152,9 +152,59 @@ job "email-blast" {
 }
 ```
 
+### Interactions with [`periodic`][periodic] 
+
+When trying to schedule a job that is both `parameterized` and `periodic`, an internal 
+hierarchy comes into play: A parametrized job should not be dispatched without 
+parameters by the periodic configuration, forcing the parameterized option to take
+precedence: Once the job is dispatched and given parameters, the 
+periodic configuration comes into play and a new jobs will be dispatched according 
+to the periodic configurations and with the parameters of the triggering job.
```

Comment:
````suggestion
Nomad uses an internal hierarchy when scheduling a job that is both `parameterized` and [`periodic`][periodic].

```plaintext
parameterized => periodic => batch
```

1. Nomad does not dispatch a periodic job with null parameters in the periodic configuration. This forces the parameterized job to take precedence over the periodic job.
1. After Nomad dispatches the parameterized job and gives it parameters, Nomad uses the periodic configuration.
1. Nomad dispatches new jobs according to the periodic configuration that uses thee parameters from the triggering parameterized job.
````

I reorganized based on Daniel's L164 suggestion. The opening paragraph contains a lot of information, and I had to read it a couple of times to understand it all. I tried to reorganize into a basic sentence structure and numbered list so the content is easier to scan.

Style guide -> present tense, active voice, direct language (see https://github.com/hashicorp/team-nomad/blob/main/wiki/docs.md#sentence-structure)

Reply:
@Juanadelacuesta did you want to include this suggestion?